### PR TITLE
feat: determine dependencies of npm projects

### PR DESCRIPTION
**Pull request by Ensar025**

### CODE REVIEW

File: packages/npm-dependency-fetcher/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "version": "1.0.0",
+  "description": "NPM dependency fetcher that is able to fetch in depth",
+  "main": "dist/index.js",
+  "type": "module",
+  "scripts": {
+    "build": "tsc",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (vanhoofmaarten) [inline packages/npm-dependency-fetcher/package.json:12]: Change to MIT licence

File: packages/npm-dependency-fetcher/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
+{
```
Comment (vanhoofmaarten) [inline packages/npm-dependency-fetcher/tsconfig.json:1]: Can you remove the comments?

File: packages/backend/serverless/index.mts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { fetchDependenciesForPackageInDepth } from '@dependency-health-monitor/npm-dependency-fetcher';
+import type { AzureFunction, Context, HttpRequest } from '@azure/functions';
+
+const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
+  if (!req.body || !req.body.packageName) {
+    context.res = {
+      status: 401,
```
[the remark this answers is not in the document]
Comment (Ensar025) [reply packages/backend/serverless/index.mts]: Done d2f9cb68811e2e84f03ddd663020cec701349119

File: packages/npm-dependency-fetcher/tsconfig.json (hunk 2 above)
Comment (Ensar025) [reply packages/npm-dependency-fetcher/tsconfig.json:1]: Done 3df71cd4fe3ef429d251c06b87f50af6bf70ee2b

File: packages/npm-dependency-fetcher/package.json (hunk 1 above)
Comment (Ensar025) [reply packages/npm-dependency-fetcher/package.json]: Done d8a01010c24c9eb7c28e94ccac8176b5f7b80b0f

**Pull request merged**